### ISSUE TITLE
New medication added

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10487]
**Preferred term label:**
New medication added in recent weeks

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0555964
Certain conditions can arise from reactions to a vast number of different medications, so this is a newness finding rather than one for any particular drug

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Acute eosinophilic pneumonia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000706
- label: New medication added in recent weeks
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: 
- def: Medical history of starting a new medication recently.
- comment: Certain conditions can arise from reactions to a vast number of different medications, so this is a newness finding rather than one for any particular drug.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10487
- synonyms:
